frontend/src/main/java/cz/cuni/mff/xrg/odcs/frontend/gui/views/PipelineEdit.java

```diff
@@ -908,7 +932,9 @@ public void textChange(FieldEvents.TextChangeEvent event) {
         pipelineSettingsLayout.addComponent(pipelineDescription, 1, 1);
 
         Label visibilityLabel = new Label(Messages.getString("PipelineEdit.visibility"));
-        pipelineSettingsLayout.addComponent(visibilityLabel, 0, 2);
+        if (!"organization".equals(appConfig.getString(ConfigProperty.OWNERSHIP_TYPE))) {
+            pipelineSettingsLayout.addComponent(visibilityLabel, 0, 2);
```

Comment:
What is the purpose of this line? Why is the visibilityLabel hidden for organization mode?